src/pkg/filters/filters.go

```diff
@@ -24,21 +27,48 @@ const (
 	Fails
 	// passthrough for the target
 	IdentityValue
-	// "foo" is a prefix of "foo/bar/baz"
+	// "foo" is a prefix of "foobarbaz"
 	TargetPrefixes
+	// "foo" equals any complete element prefix of "foo/bar/baz"
+	TargetPathPrefix
+	// "foo" equals any complete element in "foo/bar/baz"
+	TargetPathContains
 )
 
 func norm(s string) string {
 	return strings.ToLower(s)
 }
 
+// normPathElem ensures the string is:
+// 1. prefixed with a single os path delimiter (ex: `/`)
+// 2. suffixed with a single os path delimiter (ex: `/`)
```

Comment:
nit: saying `os path delimiter` makes it sound platform specific. This *should not* be platform specific, it should always be `/`
```suggestion
// 1. prefixed with a single path delimiter (ex: `/`)
// 2. suffixed with a single path delimiter (ex: `/`)
```